### ISSUE TITLE
setMathControlState is not implemented on Mac, Linux, or X64 Windows

### ISSUE THREAD

**Issue by jamesu** [GarageGames/Torque3D#1179]
Platform::setMathControlState does not appear to be implemented for Mac, Linux, or X64 Windows. This function turns out to be very important as otherwise you will get weird rounding errors on floating point operations during execution.

**Comment by jamesu** [comment]
Still not implemented it for x64 windows, though you should be able to use the same asm. Unless there is something blocking this?

**Comment by eightyeight** [comment]
Nothing blocking :).

**Comment by LuisAntonRebollo** [comment]
@jamesu, we really need to handle this FPU changes?

We use a flag for avoid DX9 change FPU precission, and new DX or OpenGL dont change precission.

**Comment by jamesu** [comment]
@LuisAntonRebollo yes, since the state of the FPU is not guaranteed across platforms (AFAIK). For instance on linux we had big problems with the precision of a float in an AI calculation causing a crash. Best to be explicit IMO ;)

**Comment by LuisAntonRebollo** [comment]
Ok,thx for info :)

Re-opened for future research.
